test/parallel_api/numeric/numeric.ops/reduce_by_segment.pass.cpp

```diff
@@ -368,10 +368,12 @@ run_test()
     std::cout << "\t" << "run_test<float,  BinaryPredicate<float>,  BinaryOperation<float>>();" << std::endl;
 #endif
     run_test<float,  BinaryPredicate<float>,  BinaryOperation<float>>();
+#ifndef _PSTL_ICPX_TEST_RED_BY_SEG_BROKEN_64BIT_TYPES
```

Comment:
Replace
```C++
#ifndef _PSTL_ICPX_TEST_RED_BY_SEG_BROKEN_64BIT_TYPES
```
to
```C++
#if !_PSTL_ICPX_TEST_RED_BY_SEG_BROKEN_64BIT_TYPES || ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION
```

Reply:
Done. I have also removed the macro exclusion of flag predicate testing. This test takes the safe codepath due to the predicate, so we do not need to exclude it here.